component-samples/demo/db/README.md

```diff
@@ -15,6 +15,9 @@ The following are the system requirements for FDO DB.
 
 The FDO DB can be executed as a docker service only. 
 
+
+***NOTE***: If MariaDB's `latest` or `lts` tag is pulling 11.* version. Make sure to remove the deprecated configuration variables in `custom/config-file.cnf`.
+
```

Comment:
Should we add the list of deprecated config variables here?

Reply:
The list keeps on changing. So added the link to MariaDB system variable documentation.